vocab/.gitignore

```diff
@@ -0,0 +1 @@
+rdf/*.ttl
```

Comment:
I just noticed we keep ontology.ttl in rdf directory,
I'll move it (either in ./vocab or ./model) and the run.sh will be responsible to put it there.

Reply:
That's done